main.py

```diff
@@ -398,6 +398,10 @@ def update(self, request: Request) -> JSONResponse:
             if enable is True:  # enable if inactive
                 with evc.lock:
                     evc.deploy()
+            elif evc.is_enabled() and redeploy:
+                with evc.lock:
+                    evc.remove_current_flows()
```

Comment:
I did more digging and `failover_flows` flows are not left behind. This is because if `dynamic_backup_path` is True and the EVC tries to re-deploy deleting the previous flows. To summarize, the difference is [here](https://github.com/kytos-ng/mef_eline/blob/fad47df3a2c182c00bfd5f8e749c9881f2ec93b7/models/evc.py#L593-L594) and `dynamic_backup_path` value.